draft-ietf-oauth-identity-chaining.md

```diff
@@ -155,6 +174,7 @@ audience
 
 * If the request itself is not valid or if the given resource or audience are unknown, or are unacceptable based on policy, the authorization server MUST deny the request.
 * The authorization server MAY add, remove or change claims. See [Claims transcription](#claims-transcription).
+* If the inbound token is sender constrained and the resource server is acting as the Client, the Client MUST provide proof of possession of its private key, using either mTLS or DPoP, in the token exchange request to AS-A in Step (B). This is not applicable to the authorization server acting as the Client since it performs an "internal token exchange".
```

Comment:
I'm admittedly a bit confused at present but I think the "the inbound token is sender constrained and the resource server is acting as the Client, the Client MUST provide proof of possession of its private key, using either mTLS or DPoP, in the token exchange request to AS-A in Step (B)" part isn't possible?